plugins/auth/auth_proxy.js

```diff
@@ -103,8 +113,8 @@ exports.try_auth_proxy = function (connection, hosts, user, passwd, cb) {
         response.push(rest);
         if (cont !== ' ') return;
 
-        let key;
-        let cert;
+        const key = self.config.get(self.tls_cfg.main.key || 'tls_key.pem', 'binary');
+        const cert = self.config.get(self.tls_cfg.main.cert || 'tls_cert.pem', 'binary');
```

Comment:
I'm kinda of torn on this change. The variable declarations need to be in this scope, but this change will load the (cached) TLS certificates, even if the next SMTP server never advertises `STARTTLS`. Is there some benefit to loading the certs up here?

Reply:
You're right I just wanted to prevent loading files several times in the loop. I'll return them back to the prev place as you mentioned they're cached. Right?

Reply:
Yep, the config loader caches the config files.

Reply:
Done